deployments/data8x/config/common.yaml

```diff
@@ -63,6 +63,9 @@ jupyterhub:
           - b1b001615f9eb7254387f667569e39fb
           - c741d11d7f0bed9930e04276115b1a37
           - fd5ad8c476846392d8dfb1765d1a6ac5
+          # 2813 Eric Van Dusen
+          - ericvd
```

Comment:
Their EdX IDs needed to be added here - they look like the random strings you see above. You can find yours by:

1. Enrolling in the class
2. Launching into the lab
3. Finding the user id from control panel